app/helpers/storage_helper/textual_summary.rb

```diff
@@ -57,7 +57,7 @@ def textual_total_space
 
   def textual_uncommitted_space
     return nil if @record["total_space"].nil?
-    space = if @record["uncommitted"].blank? || @record["uncommitted"] == ""
+    space = if @record['uncommitted'].blank?
```

Comment:
Why the change in quotes?  The former was fine and in fact was consistent with all of the code around it.  (Not required to fix, just not sure why you did it).